cumulusci/cli/org.py

```diff
@@ -258,9 +259,9 @@ def calculate_org_days(info):
 def org_info(runtime, org_name, print_json):
     org_name, org_config = runtime.get_org(org_name)
     org_config.refresh_oauth_token(runtime.keychain)
-
+    console = Console()
     if print_json:
-        click.echo(
+        rich.print_json(
```

Comment:
There's an open issue at willmcgugan/rich#1591. Should we hold off?

Reply:
I feel ok because it looks like it's only an issue when piping to other things (which I know you'd probably be bummed about). However, printing of JSON itself seems to be working. What are your thoughts?

Reply:
Since piping  machine-readable output to `jq` and other tools is the primary use case  for this option, I'm leaning towards being conservative. (That said, piping to jq is already broken because we're logging to stdout.)

---

```console
$ cci org info --json dev | tail -n +2 | jq
parse error: Invalid string: control characters from U+0000 through U+001F must be escaped at line 3, column 3
```

Same command works on main. The `access_token` has a length of 133 characters (including indent), while `print_json` truncates this line to the width of my terminal, 107 characters.

Reply:
Ah, never mind, `console.print(json_str, soft_wrap=True)` fixes truncation. I'll push a fix upstream.

Reply:
Odd, this command works for me on both this branch and main. I'm seeing my access token with 113 chars on each command.

Reply:
Sure, but how wide is your terminal? Repro:
```python-console
>>> from rich import print_json, get_console
>>> import json
>>> test_dict = {'access_token': '................................................................................................................', 'scratch': 100, 'set_password': False}
>>> c = get_console()
>>> c.width
107
>>> rich.print_json(json.dumps(test_dict))
{
  "access_token": "........................................................................................
  "scratch": 100,
  "set_password": false
}
```

Reply:
Ahh, ok I can reproduce that with a small window. I'll go back to printing json with `click.echo()` for the time being and we can keep an eye on the bug.